torch/quantization/observer.py

```diff
@@ -214,7 +215,8 @@ def _calculate_qparams(self, min_val, max_val):
             )
 
         qmin, qmax = self._calculate_qmin_qmax()
-
+        orig_min = min_val
+        orig_max = max_val
         min_val = torch.min(min_val, torch.zeros_like(min_val))
         max_val = torch.max(max_val, torch.zeros_like(max_val))
```

Comment:
this code was a bit confusing (before this PR).  Maybe we can rename these something like `min_val_neg` and `max_val_pos` in the rest of the function?